api-2.0/src/test/java/org/openmrs/module/attachments/obs/TestHelper.java

```diff
@@ -0,0 +1,362 @@
+package org.openmrs.module.attachments.obs;
+
+import liquibase.util.file.FilenameUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.RandomStringUtils;
+import org.openmrs.*;
```

Comment:
Please avoir wildcard imports.
@chinzou could you direct @ridmal into configuring IntelliJ for not doing so?

Reply:
@ridmal 
to avoid using wildcard imports in Intellij.
go to : File > Settings , Then Editor > Code Style > Java > Imports
you'll find `Class count to use import with '*':` property, just set it to a very high value. like 999 :p
more details [here](https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports)

Reply:
Ok , Thank you. @chinzou and @mks-d